regarding building openthread in native pc

Hi all,
I am new to thread, i wanted to test CLI application on my linux PC, for that i have followed README in CLI folder, where after "make -f examples/Makefile-posix" command, it is saying build for node1, node2, for which a posix folder should be generated in above command i am unable to view this folder. Can any one help me in this regard and is it possible to test in native PC if so please guide me.

Thanks,
Praveen Kumar k

hi @jwhui , I think it might not only about the doc.
Under my environment(4.2.0-38-generic #45~14.04.1-Ubuntu), I have to use sudo before 'make -f examples/Makefile-cc2538' or 'make -f examples/Makefile-posix' or else there would be 'Permission denied' issue; and the binaries would be in the absolute path '/output/bin/arm-none-eabi-ot-cli' and '/output/x86_64-unknown-linux-gnu/bin/ot-cli' respectively.

` make -f examples/Makefile-cc2538 
  BUILD    cortex-m3-arm-none-eabi
make -C build/cortex-m3-arm-none-eabi --no-print-directory all
  CHECK    .local-version
/bin/sh: .local-version.N: Permission denied
make[1]: *** [.local-version] Error 1
make: *** [do-build-cortex-m3] Error 2

make -f examples/Makefile-posix
  BUILD    x86_64-unknown-linux-gnu
make -C build/x86_64-unknown-linux-gnu --no-print-directory all
  CHECK    .local-version
/bin/sh: .local-version.N: Permission denied
make[1]: *** [.local-version] Error 1
make: *** [do-build-x86_64-unknown-linux-gnu] Error 2
`

--- a/examples/apps/cli/README.md
+++ b/examples/apps/cli/README.md
@@ -19,7 +19,7 @@ $ make -f examples/Makefile-posix
 Spawn the process:
 
 ```bash
-$ cd openthread/examples/posix/app/cli
+$ cd <path-to-openthread>/output/<platform>/bin
 $ ./ot-cli 1
 ```
 
@@ -54,7 +54,7 @@ Done
 Spawn the process:
 
 ```bash
-$ cd openthread/examples/posix/app/cli
+$ cd <path-to-openthread>/output/<platform>/bin
 $ ./ot-cli 2
 ```
 

